sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/PiiEntityCollection.java

```diff
@@ -11,17 +11,21 @@
  */
 @Immutable
 public final class PiiEntityCollection extends IterableStream<PiiEntity> {
-
+    private final String redactedText;
     private final IterableStream<TextAnalyticsWarning> warnings;
 
     /**
      * Creates a {@link PiiEntityCollection} model that describes a entities collection including warnings.
      *
      * @param entities An {@link IterableStream} of {@link PiiEntity Personally Identifiable Information entities}.
+     * @param redactedText The text of the input document with all of the PII information redacted out. Only returned
+     * for API version v3.1-preview.2 and up.
```

Comment:
Same with all the comments. Not need to say that it will only be returned for API v3.1-preview.2 if in reality it will only be included in out 5.1.0-beta.1  library.
I suggest removing it